python/xorbits/_mars/dataframe/groupby/core.py

```diff
@@ -498,6 +505,8 @@ def execute(cls, ctx, op: "DataFrameGroupByOperand"):
             else:
                 df = inp
                 by = op.by
+            if not op.is_list_by and isinstance(by, list):
```

Comment:
In what scenario does this happen?

Reply:
`by` converts to list internally when calling groupby, Pandas 2.0 has some slight differences between scalar and list, here add a operand property to record if is a list.